pypit/spectrographs/lris.py

```diff
@@ -12,8 +13,74 @@
 from astropy.io import fits
 
 from pypit import msgs
-from pypit.arparse import load_sections
+from pypit import arparse
 from pypit import ardebug as debugger
+from. import spectroclass
```

Comment:
Does this work? In any case, we should try to keep things consistent, so how about: `from pypit.spectrographs import spectroclass`

Reply:
modified (it does work)